segfast/segyio_loader.py

```diff
@@ -112,8 +108,9 @@ def load_headers(self, headers, reconstruct_tsf=True, sort_columns=True, tracewi
 
         Parameters
         ----------
-        headers : sequence
-            Names of headers to load.
+        headers : sequence or dict
```

Comment:
Usually we work with a case when most headers are stored according to the standard, but only some of them are not. This case is pretty hard to handle using current parametrization: you have to pass a dict with manually defined starting bytes for each header.

I'd propose the following parametrization: `headers` is an array-like where each element can be:
1. `str` -- header name,
2. `int` -- header starting byte,
3. `TraceHeader` -- used as is,
4. `tuple` -- args to init `TraceHeader`,
5. `dict` -- kwargs to init `TraceHeader`.

This will allow writing something like:
`headers=["FieldRecord", "TraceNumber", "offset", ("FirstBreak", 81), {"name": "Horizon1", "start_byte": 20, "dtype": ">f4"}]`

Also, @Pennyroyall begged to add `"all"` keyword to load everything standard.